Configuration/ProcessModifiers/python/vertexTrimming_cff.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier is used to enable vertex trimming in the tracking sequences
+# and in all related reconstruction modules
+vertexTrimming = cms.Modifier()
```

Comment:
can it be renamed to signal it's about the phase-2 HLT?

Reply:
I renamed it to phase2_hlt_vertexTrimming in c05a4f2ff49a379ba766216d9e1151848535f7fd